modular_skyrat/master_files/code/modules/mob/living/simple_animal/friendly/poppy.dm

```diff
@@ -1,10 +1,11 @@
 /mob/living/simple_animal/pet/poppy
-	name = "Poppy the Safety Possum"
+	name = "Poppy the Safety Inspector"
```

Comment:
Change it back, the name is because it half-rhymes with Possum.

Reply:
Another undocumented change.